django/contrib/contenttypes/fields.py

```diff
@@ -131,7 +131,7 @@ def _check_content_type_field(self):
                 )
             ]
         else:
-            if not isinstance(field, models.ForeignKey):
+            if not field.many_to_one:
                 return [
                     checks.Error(
                         "'%s.%s' is not a ForeignKey." % (
```

Comment:
Did you not revise the messages (ForeignKey -> "foreign key") here for any reason?